perception/tensorrt_classifier/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.5)
 project(tensorrt_classifier)
 
+# cspell: ignore fopenmp
```

Comment:
```suggestion
```
`fopenmp` is added in <https://github.com/tier4/autoware-spell-check-dict/pull/574>.